cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,631 @@
+from widgetastic_manageiq import V2VPaginatorPane
+
+
+class MigrationView(BaseLoggedInPage):
+    title = Text("#explorer_title_text")
+
+    @property
+    def in_explorer(self):
+        nav_menu = VersionPicker(
+            {
+                Version.lowest(): ["Compute", "Migration"],
+                "5.10": ["Compute", "Migration", "Migration Plans"],
+            }
+        )
+
+        return self.logged_in_as_current_user and self.navigation.currently_selected == nav_menu
+
+    @property
+    def is_displayed(self):
+        return self.in_explorer
+
+
+class DashboardCards(View):
+    not_started_plans = MigrationDashboardStatusCard("Not Started")
+    in_progress_plans = MigrationDashboardStatusCard("In Progress")
+    completed_plans = MigrationDashboardStatusCard("Complete")
+    archived_plans = VersionPicker(
+        {
+            Version.lowest(): MigrationDashboardStatusCard("Infrastructure Mappings"),
+            "5.10": MigrationDashboardStatusCard("Archived"),
+        }
+    )
+
+    def read(self):
+        return [c.read().get("name") for c in self.sub_widgets if c.is_active]
+
+
+class MigrationPlanView(MigrationView):
+    dashboard_cards = View.nested(DashboardCards)
+    plans_list = ConditionalSwitchableView(reference="dashboard_cards")
+    paginator_view = View.include(V2VPaginatorPane, use_parent=True)
+    title = Text('.//div[contains(@class, "pull-left")]//h3')
+    create_migration_plan = Text(locator='(//a|//button)[text()="Create Migration Plan"]')
+    configure_providers = Text(locator='//a[text()="Configure Providers"]')
+    sort_type_dropdown = SelectorDropdown("id", "sortTypeMenu")
+    sort_direction = Text(locator=".//span[contains(@class,'sort-direction')]")
+    # TODO - fix this once dev fixes the locator
+    progress_card = MigrationProgressBar(locator=".//div[3]/div/div[3]/div[3]/div/div")
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
+    # 5.9 field
+    migr_dropdown = MigrationDropdown(text="Not Started Plans")
+
+    @property
+    def in_migration_plan(self):
+        return (
+            len(self.browser.elements('.//div[contains(@class,"card-pf")]')) > 0
+            and len(self.browser.elements(PFIcon.icons.WARNING)) < 1
+        )
+
+    @property
+    def is_displayed(self):
+        return self.in_migration_plan
+
+
+class NotStartedPlansView(MigrationPlanView):
+    TITLE = "Not Started"
+    plans_not_started_list = MigrationPlansList("plans-not-started-list")
+
+    @property
+    def is_displayed(self):
+        def in_migration():
+            return self.in_migration_plan and self.plans_not_started_list.is_displayed
+
+        in_view = VersionPicker(
+            {
+                Version.lowest(): in_migration,
+                "5.10": in_migration and self.TITLE in self.title.read(),
+            }
+        ).pick()
+        return in_view
```

Comment:
for 5.9, this has to be called as a function, in 5.10 it is a property.

5.9:
```
In[2]: rhv = providers.get_crud('rhv42')
In[3]: migration_plan_collection = appliance.collections.v2v_migration_plans
In[4]:  migration_plan = migration_plan_collection.instantiate(name='testing_juju')
In[5]: view = navigate_to(migration_plan, 'NotStarted')
[WARNING] [py.warnings] /home/juwatts/repos/cfme_venv/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
 (/home/juwatts/repos/cfme_venv/lib/python2.7/site-packages/urllib3/connectionpool.py:847)
In[6]: view.is_displayed
Out[6]: <function cfme.v2v.migration_plans.in_migration>
In[7]: view.is_displayed()
Out[7]: True

```

5.10
```
In[2]: rhv = providers.get_crud('rhv42')
In[3]: migration_plan_collection = appliance.collections.v2v_migration_plans
In[4]:  migration_plan = migration_plan_collection.instantiate(name='jujuju')
In[5]: view = navigate_to(migration_plan, 'NotStarted')
[WARNING] [py.warnings] /home/juwatts/repos/cfme_venv/lib/python2.7/site-packages/urllib3/connectionpool.py:847: InsecureRequestWarning: Unverified HTTPS request is being made. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning)
 (/home/juwatts/repos/cfme_venv/lib/python2.7/site-packages/urllib3/connectionpool.py:847)
In[6]: view.is_displayed()
Traceback (most recent call last):
  File "/home/juwatts/repos/cfme_venv/lib/python2.7/site-packages/IPython/core/interactiveshell.py", line 2878, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-6-98cddd87cc40>", line 1, in <module>
    view.is_displayed()
TypeError: 'bool' object is not callable
In[7]: view.is_displayed
Out[7]: True
```